core/shared/src/main/scala/zio/Runtime.scala

```diff
@@ -174,4 +200,59 @@ object Runtime {
     val environment = r
     val platform    = platform0
   }
+
+  lazy val default = Runtime((), Platform.default)
+
+  lazy val global = Runtime((), Platform.global)
+
+  /**
+   * Unsafely creates a `Runtime` from the default platform and a `ZLayer`
+   * whose resources will be allocated immediately, and not released until the
+   * `Runtime` is shut down or the end of the application.
+   *
+   * This method is useful for small applications and integrating ZIO with
+   * legacy code, but other applications should investigate using
+   * [[ZIO.provideLayer]] directly in their application entry points.
+   */
+  def unsafeDefault[R <: Has[_]](layer: ZLayer.NoDeps[Any, R]): Runtime.Managed[R] =
+    unsafeMake(layer, default.platform)
+
+  /**
+   * Unsafely creates a `Runtime` from Scala's global execution context and a
+   * `ZLayer` whose resources will be allocated immediately, and not released
+   * until the `Runtime` is shut down or the end of the application.
+   *
+   * This method is useful for small applications and integrating ZIO with
+   * legacy code, but other applications should investigate using
+   * [[ZIO.provideLayer]] directly in their application entry points.
+   */
+  def unsafeGlobal[R <: Has[_]](layer: ZLayer.NoDeps[Any, R]): Runtime.Managed[R] =
```

Comment:
How about we go down to a single method and call it `unsafeFromLayer`, with `platform` as an argument that defaults to `Platform.default`?

Reply:
Yes I like that a lot. Will make that change.